Correct more type/member signatures in Annex C, "Standard library" (#1480)
dotnet/csharpstandard: standard/general-description.md

This specification is divided into the following subdivisions: front matter; language syntax, constraints, and semantics; and annexes.

> Examples are provided to illustrate possible forms of the constructions described, and some might make use of types and members that are outside this specification. References are used to refer to related clauses and subclauses. Notes are provided to give advice or guidance to implementers or programmers. Annexes provide additional information and summarize the information contained in this specification.

Reviewer: ```suggestion
Examples are provided to illustrate possible forms of the constructions described, and some make use of types and members that are outside this specification. References are used to refer to related clauses and subclauses. Notes are provided to give advice or guidance to implementers or programmers. Annexes provide additional information and summarize the information contained in this specification.
```

> Examples are provided to illustrate possible forms of the constructions described, and they might make use of types and members that are supported by .NET, but which are not required by this specification. References are used to refer to related clauses and subclauses. Notes are provided to give advice or guidance to implementers or programmers. Annexes provide additional information and summarize the information contained in this specification.

Reviewer 2: Maybe find wording to remove “.NET”, part of the rationale for a Standard is to move away from “C# is a .NET language” so it is better to reduce references to it. I'll offer the following suggestion:

```suggestion
Examples are provided to illustrate possible forms of the constructions described, and some might make use of types and members that are outside this specification. References are used to refer to related clauses and subclauses. Notes are provided to give advice or guidance to implementers or programmers. Annexes provide additional information and summarize the information contained in this specification.
```
